packages/ui/src/theme/css/component/select.scss

```diff
@@ -25,14 +25,18 @@
   option {
```

Comment:
Currently Chrome and Edge on Windows don't show the disabled color for a disabled option. Can we fix that too? Applying the same fix here *should* work (and then maybe we don't need the firefox specific fix) though we should probably do a little more rigorous testing since there are always issues setting the color property for options/select cross browser :(

Reply:
Looks good!

Screenshots from Windows in my own Browserstack account on latest of each browser. I guess this is a data point for needing automation for this :)

**Edge Dark**
<img width="976" alt="EdgeDark" src="https://user-images.githubusercontent.com/376920/182457970-ea585960-70d0-4025-a720-c409fc2097ee.png">
**Edge Light**
<img width="985" alt="EdgeLight" src="https://user-images.githubusercontent.com/376920/182457971-2e11f1c0-609c-4248-bf24-45ff56f49680.png">
**FF Dark**
<img width="994" alt="FF Dark" src="https://user-images.githubusercontent.com/376920/182458170-4997d2c7-d19d-4ab9-a128-8518cd472317.png">
**FF Light**
<img width="965" alt="FF Light" src="https://user-images.githubusercontent.com/376920/182458173-8d4840fa-0d5d-4202-8123-d1bda7de15d9.png">
**Chrome Dark**
<img width="969" alt="Chrome Dark" src="https://user-images.githubusercontent.com/376920/182458709-1b69db56-4a68-437b-982b-5239df8519fa.png">
**Chrome Light**
<img width="957" alt="Chrome  Light" src="https://user-images.githubusercontent.com/376920/182458711-033d2e58-ebca-4b51-a959-56e2bd03971f.png">

Reply:
Thank you for testing these @hbuchel! Definitely a datapoint for needing cross-platform automated visual regression tests.